src/applications/vaos/actions/newAppointment.js

```diff
@@ -160,6 +167,28 @@ export function openTypeOfCarePage(page, uiSchema, schema) {
   };
 }
 
+export function fetchFacilityDetails(facilityId) {
+  let facilityDetails;
+
+  return async dispatch => {
+    dispatch({
+      type: FORM_FETCH_FACILITY_DETAILS,
+    });
+
+    try {
+      facilityDetails = await getFacilityInfo(facilityId);
+    } catch (e) {
```

Comment:
I think we should add a Sentry.captureException call in here so we don't swallow the error.